src/Sentry.AspNetCore/SentryAspNetCoreOptions.cs

```diff
@@ -39,6 +39,16 @@ public class SentryAspNetCoreOptions : SentryLoggingOptions
         /// </summary>
         public TimeSpan FlushTimeout { get; set; } = TimeSpan.FromSeconds(2);
 
+        /// <summary>
+        /// Controls whether the case of the standard (Production, Development and Staging) EnvironmentName supplied by HostingEnvironment
```

Comment:
```suggestion
        /// Controls whether the casing of the standard (Production, Development and Staging) environment name supplied by <see cref="Microsoft.AspNetCore.Hosting.HostingEnvironment" />
```

Reply:
Hmm, VS complains with this:
```
XML comment has cref attribute 'HostingEnvironment' that could not be resolved	
Sentry.AspNetCore (netcoreapp3.0), Sentry.AspNetCore (netstandard2.0)
```

Reply:
A reference to `Microsoft.AspNetCore.Hosting.IHostingEnvironment` (which is more accurate anyway) worked, so I'll go with that.